utility/utils.py

```diff
@@ -966,3 +966,39 @@ def get_random_str(size=13):
     """
     chars = string.ascii_lowercase + string.digits
     return ''.join(random.choice(chars) for _ in range(size))
+
+
+def run_async(command):
+    """
+    Run command locally and return without waiting for completion
+
+    Args:
+        command (str): The command to run.
+
+    Returns:
+        An open descriptor to be used by the calling function.
+
+    Example:
+        command = 'oc delete pvc pvc1'
+        proc = run_async(command)
+        ret, out, err = proc.async_communicate()
+    """
+    log.info(f"Executing command: {command}")
+    popen_obj = subprocess.Popen(
```

Comment:
you have to return this object for it to be meaningful for tests to later communicate with it.

Reply:
popen_obj is the return value of this method.